haskell/test/Lorentz/Contracts/Test/Permit.hs

```diff
@@ -370,6 +368,15 @@ permitSpec originate = do
               callPermit stablecoinContract wallet1PK wallet1SK 0 Unpause
               lCallEP stablecoinContract (Call @"Unpause") () `catchExpectedError` mgmNotPauser
 
+      specify "pauser does not consume permits" $
```

Comment:
There are two "pauser does not consume permits" tests, is that intended?

Reply:
> is that intended

Yeah, they're under two different `describe` groups: one for the `pause` entrypoint, one for the `unpause` entrypoint.
I've renamed them to make the difference stand out.